src/strategy-heat-capacitor-functions.js

```diff
@@ -137,10 +140,21 @@ function calculateSchedule(
     boostTempCool: boostTempCool,
     heatingDuration: buyDuration,
     coolingDuration: sellDuration,
+    minimalSchedule: [] //array of dicts with date as key and temperature as value
   };
 
+  function pushTempChange(startDate, minutes, tempAdj, sp){
+    if ((schedule.minimalSchedule.length>0) && (schedule.minimalSchedule.at(-1).adjustment == tempAdj)) return;
```

Comment:
We cannot use the array.at() function, as it requires node 16.6. We should support node 14.0.

Reply:
Used .slice(-1)[0] instead.

Reply:
Or you could just use `schedule.minimalSchedule[schedule.minimalSchedule.length - 1].adjustment` instead of making a copy with `slice`.

Reply:
Hehe... I will change it. I did consider using .length, but the repetition of "schedule.minimalSchedule" hurts my eyes :-) But .slice is also quite unintuitive and not pretty from an algorithm point of view.